app/assets/javascripts/blacklight/facet_load.js

```diff
@@ -0,0 +1,14 @@
+//= require blacklight/core
+(function($) {
+  Blacklight.onLoad(function() {
+    // adjust width of facet columns to fit their contents
+    $('ul.facet-values, ul.pivot-facet').map(function(){
+      function longer (a,c){return a.textContent.length>c.textContent.length?a:c;}
+      var longest = Array.prototype.reduce.apply($(this).find('.facet-count span'),[longer]);
```

Comment:
Do we have an IE version that we explicitly don't support? This will break on IE8

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce